vsphere/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Fixed***:
```

Comment:
Why is it Fixed instead of Added?

Reply:
I left it as fixed because the feature should be able to apply to the metric filters already, but I don't mind updating it to added